proto/frequenz/api/microgrid/microgrid.proto

```diff
@@ -219,6 +220,9 @@ service Microgrid {
   // * Battery: Checks if DC relays are open, then
   //  * closes DC relays
   //
+  // * Relays:
```

Comment:
is the order o actions important here?

Reply:
Also just for my understanding. Where is a specific relay linked to a specific battery-inverter instance?

Reply:
>Also just for my understanding. Where is a specific relay linked to a specific battery-inverter instance?

It would be a predecessor to components, e.g., inverters. The docs in the relay module should specify this already.